ulid_test.go

```diff
@@ -278,6 +278,15 @@ func TestTimestamp(t *testing.T) {
 	if got, want := ulid.Timestamp(tm), uint64(1000); got != want {
 		t.Errorf("for %v, got %v, want %v", tm, got, want)
 	}
+
+	mt := ulid.MaxTime()
+	dt := time.Unix(int64(mt/1000), int64((mt%1000)*1000000)).Truncate(time.Millisecond)
+	t.Logf("MaxTime=%d = %v", mt, dt)
+	ts := ulid.Timestamp(dt)
+	t.Logf("ts=%d", ts)
+	if ts-mt > 1000 || mt-ts > 1000 {
```

Comment:
With that, I'd remove the above logging calls and add the following, keeping it consistent with the rest of the tests.

```go
if got, want := ts, mt; got != want {
    t.Errorf("got timestamp %d, want %d", got, want)
}
```